src/components/Hero.jsx

```diff
@@ -0,0 +1,161 @@
+import { PersonalizedText } from './shared/typography';
+import { Paragraph } from './shared/typography';
+
+const Hero = () => {
+	return (
+		<>
+			<div className='flex flex-col md:flex-row'>
+				<div className='w-[100%] md:w-1/2 '>
+					<div className='h-1/3 flex flex-col justify-end ml-2' style={{ color: '#FF4409' }}>
+						<PersonalizedText>Welcome family...</PersonalizedText>
+					</div>
+					<div className='h-1/3'>
+						<Paragraph style={{ fontSize: '75px', fontWeight: '500', letterSpacing: '0.75px', lineHeight: 'normal' }}>
+							Lorem Ipsum Dolar sit Amet Luyto
+						</Paragraph>
+					</div>
+					<div className='h-1/4'>
+						<button
+							className='px-4 py-2 rounded-md border border-solid border-black bg-orange-500 shadow-button mt-20 md:mt-35'
+							style={{
+								boxShadow: '2px 2px 0px 0px #000, 3px 4px 9.2px 0px rgba(222, 222, 222, 0.48) inset',
+								fontFamily: 'Syne, sans-serif',
+								fontWeight: '700',
+								marginLeft: '20px',
+							}}>
+							Register Now
+						</button>
+					</div>
+				</div>
+				<div
+					className='w-[100%] md:w-1/2  flex items-center justify-center relative '
+					style={{ height: '600px', marginLeft: '29px' }}>
+					<div
+						className='absolute w-[201px] h-[270px] md:w-[303.075px] md:h-[404.558px]'
+						style={{
+							borderRadius: '6px',
+							border: '1px solid #000',
+							background: '#FFF',
+							boxShadow: '2px 3px 0px 0px #000',
+							// top: '100px',
+							right: '42%',
+							transform: 'rotate(-7.121deg)',
+							flexShrink: 0,
+							marginRight: '20px',
+							zIndex: 1,
+						}}>
+						<div
+							className='md:w-[275px] md:h-[320px] w-[188px] h-[221px] md:mt-[12px] mt-[6px] md:ml-[12px] ml-[6px]'
+							style={{
+								backgroundImage: "url('https://res.cloudinary.com/dkpdi1wek/image/upload/v1709662790/left_vv4m1m.jpg')",
+								backgroundSize: 'cover',
+								backgroundPosition: 'center',
+								borderRadius: '4px',
+								// height: 'calc(100% - 40px)',
+								// width: '275px',
+								// height: '320px',
+								// marginLeft: '12px',
+								// marginTop: '10px',
+							}}></div>
+					</div>
+					<div
+						className='absolute md:h-[414px] md:w-[300px] w-[210px] h-[278px]'
+						style={{
+							borderRadius: '6px',

```

Comment:
Can use tailwind for these things.

Reply:
fixed https://github.com/dscnitrourkela/project-udon/pull/17/commits/458135db8b03d72a81e6483df3a32f7b7154882b